### PR TITLE
fix: timelineEvent -> inlineEvent; parse .md files

### PR DESCRIPTION
**Pull request by ebullient**
Check me on this one... I think I did everything ok, but I may have messed something up w/ svelte

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -170,20 +169,9 @@ export default class Calendarium extends Plugin {
...
-        let tag = calendar.timelineTag;
-        if (this.syncTimelines(calendar)) {
-            tag = this.app.plugins
-                .getPlugin("obsidian-timelines")
-                .settings.timelineTag.replace("#", "");
-        }
-        return tag ?? calendar.timelineTag ?? "";
+    inlineEventTag(calendar: Calendar) {
+        let tag = calendar.inlineEventTag;
+        return tag ?? calendar.inlineEventTag ?? "";
```
Comment (valentine195) [inline src/main.ts:174]: Kind of redundant

File: src/settings/creator/Containers/Info.svelte [hunk 2]
```diff
@@ -177,22 +137,22 @@
...
+                value={inlineEvents}
                 on:click={() => {
-                    $calendar.supportTimelines = !$calendar.supportTimelines;
+                    $calendar.supportInlineEvents = !$calendar.supportInlineEvents;
                 }}
             />
-            {#if timelines}
-                {#key $calendar.syncTimelines}
+            {#if inlineEvents}
+                {#key $calendar.inlineEvents}
```
Comment (valentine195) [inline src/settings/creator/Containers/Info.svelte:148]: This should probably be `supportInlineEvents`

File: src/main.ts (hunk 1 above)
Comment (ebullient) [reply src/main.ts:174]: yea.. old code. I'll clean it up. ;)

File: src/settings/creator/Containers/Info.svelte (hunk 2 above)
Comment (valentine195) [reply src/settings/creator/Containers/Info.svelte:148]: This isn't resolved; `inlineEvents` does not exist on the `Calendar` interface.

Comment (valentine195) [reply src/settings/creator/Containers/Info.svelte:148]: In Svelte, a store's underlying data can be accessed using the `$` prefix.

Essentially, `$` does something like this:

```js
let store = writable(calendar);

//$store resolves to:
let $store;
store.subscribe((val) => $store = val);
```

So `$calendar` is the `Calendar` object that we created a store from.

**Pull request merged**